### PR TITLE
Reliability fix for Banana Pi M2+

### PR DESCRIPTION
**Pull request by akuker**
I knew it was something simple that I was missing!!! 

With this fix, Banana Pi M2+ seems to be much more stable, but incredibly slow.

image

Performance will be the next step. This PR should at least make the functionality more stable. (probably not perfect, but much much better)

### CODE REVIEW

File: cpp/shared/rascsi_version.cpp [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 // The following should be updated for each release
 const int rascsi_major_version = 22; // Last two digits of year
-const int rascsi_minor_version = 11; // Month
+const int rascsi_minor_version = 12; // Month
```
Comment (rdmark) [inline cpp/shared/rascsi_version.cpp:17]: Was this change deliberate here?

**Review by rdmark — APPROVED** [review]

File: cpp/shared/rascsi_version.cpp (hunk 1 above)
Comment (akuker) [reply cpp/shared/rascsi_version.cpp:17]: Ya. Since we're in December now

Comment (rdmark) [reply cpp/shared/rascsi_version.cpp:17]: Fair enough. It just seemed a bit out of place for this changeset, a better fit for a "bump version" type of PR. But never mind me. I'm just picky about these things. :)
